openedx/core/djangoapps/user_api/views.py

```diff
@@ -152,6 +154,7 @@ def post(self, request):
 
         try:
             user = create_account_with_params(request, data)
+            create_edly_user_profile(request, user)
```

Comment:
shouldn't there be a test that checks if an edly user profile is created on user registration?

Reply:
Yes, I have already added a test to checking `edly user profile` is created.